jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererConstants.kt

```diff
@@ -37,6 +37,7 @@ object CodeWhispererConstants {
     const val TOTAL_BYTES_IN_KB = 1024
     const val TOTAL_BYTES_IN_MB = 1024 * 1024
     const val TOTAL_MILLIS_IN_SECOND = 1000
+    const val TOTAL_SECONDS_IN_MINUTE: Long = 60
```

Comment:
```suggestion
    const val TOTAL_SECONDS_IN_MINUTE = 60
```
be consistent with providing types

Reply:
ack